com.unity.render-pipelines.universal/Runtime/Passes/DepthNormalOnlyPass.cs

```diff
@@ -33,7 +36,7 @@ public DepthNormalOnlyPass(RenderPassEvent evt, RenderQueueRange renderQueueRang
         /// <summary>
         /// Configure the pass
         /// </summary>
-        public void Setup(RenderTextureDescriptor baseDescriptor, RenderTargetHandle depthHandle, RenderTargetHandle normalHandle)
+        public void Setup(RenderTextureDescriptor baseDescriptor, RenderTargetHandle depthHandle, RenderTargetHandle normalHandle, RenderTargetHandle cameraDepthHandle, bool useDepthPriming)
```

Comment:
We need to make sure we do not break users. This is a public API change so could we default the new parameters to `null` and `false`?